modules/utxo-lib/test/bitgo/wallet/WalletUnspent.ts

```diff
@@ -122,17 +123,23 @@ describe('WalletUnspent', function () {
   function constructAndSignTransactionUsingPsbt(
     unspents: WalletUnspent<bigint>[],
     signer: string,
-    cosigner: string
+    cosigner: string,
+    scriptType: outputScripts.ScriptType2Of3
   ): Transaction<bigint> {
     const psbt = createPsbtForNetwork(network);
-    psbt.addOutput({
-      address: getWalletAddress(walletKeys, 0, 100, network),
-      value: BigInt(unspentSum<bigint>(unspents, 'bigint')) - BigInt(100),
-    });
+    addChangeOutputToPsbt(
+      psbt,
+      walletKeys,
+      getInternalChainCode(scriptType),
+      100,
+      BigInt(unspentSum<bigint>(unspents, 'bigint')) - BigInt(100)
```

Comment:
To be clear, here `100` is both the index used for the derivation path and also the size of the non-change output used for this test? It might be worth naming these variables so it's easier to understand how they're used.

Reply:
Yeah, duplicated from previous code, but worth improving while we're here.

Reply:
Yep, I saw the old code was doing the same thing. But just a bit confusing when `100` is used for multiple things.